### ISSUE TITLE
XMB CPU speed not applied at bootup

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MrColdbird/procfw#440]
```
What steps will reproduce the problem?
1. Set xmb cpu speed to 20/10
2. Perform reboot or vsh reset

What is the expected output? What do you see instead?

XMB runs at normal speed instead being slowed down. Slows down only after 
opening and closing VSH menu

What version of the product are you using? On what operating system?

6.20 PRO-B10.fix1, permanent patch

Please provide any additional information below.

PSP 3008 03g, all plugins disabled.
```

Original issue reported on code.google.com by `host127...@gmail.com` on 28 Apr 2012 at 9:56
- Merged into: #94
